Update window customization guide (#2411)
tauri-apps/tauri-docs: src/content/docs/plugin/window-customization.mdx

  ?.addEventListener('click', () => appWindow.close());
```

> Note that if you are using rust for frontend, you can copy code above into `<script>` tag into your `index.html`.

Reviewer: ```suggestion
Note that if you are using a Rust-based frontend, you can copy the code above into a `<script>` element in your `index.html` file.
```
